platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -150,6 +150,24 @@ typedef NS_ENUM(NSUInteger, MGLSymbolPlacement) {
     MGLSymbolPlacementLineCenter,
 };
 
+/**
+ Controls the order in which overlapping symbols in the same layer are rendered
+
+ Values of this type are used in the `MGLSymbolStyleLayer.symbolZOrder`
+ property.
+ */
+typedef NS_ENUM(NSUInteger, MGLSymbolZOrder) {
+    /**
+     Symbols will be sorted by their y-position relative to the viewport.
```

Comment:
In platform/darwin/scripts/style-spec-overrides-v8.json, we should be able to override this to say:

> Specify this z order if symbols’ appearance relies on lower features overlapping higher features. For example, symbols with a pin-like appearance would require this z order.